### DISCUSSION TITLE
Documentation looks beautiful locally but unstyled when deployed on GH pages

### DISCUSSION THREAD

**Discussion by gtauzin** [squidfunk/mkdocs-material#7773] [Q&A] [answered]
Hi,
I am trying to deploy documentation for a [public repos](https://github.com/gtauzin/kedro-dagster) with materials for mkdocs (and hopefully mike soon). I have a setup with some [customization](https://github.com/gtauzin/kedro-dagster/blob/main/mkdocs.yml) that looks good when build locally (or served). However, once I publish it to gh-pages (see [action](https://github.com/gtauzin/kedro-dagster/blob/main/.github/workflows/documentation.yml)), what I see is simply [a very raw unstyled html site](https://gtauzin.github.io/kedro-dagster/).

Note that I set my pages on gh-pages branch and /docs repository. I have tried to use /root but then my site is basically an html verison of my README.md file. mkdocs creates a /site directory. How do I get it to be the one published? 

From the action job itself, it seems like everything is happening as it should:

```
Run uv run mkdocs gh-deploy --force # TODO: Use mike to deploy versioned docs
Using CPython 3.10.12 interpreter at: /usr/bin/python3
Creating virtual environment at: .venv
Installed 139 packages in 408ms
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /home/runner/work/kedro-dagster/kedro-dagster/site
INFO    -  Documentation built in 0.28 seconds
WARNING -  Version check skipped: No version specified in previous deployment.
INFO    -  Copying '/home/runner/work/kedro-dagster/kedro-dagster/site' to 'gh-pages' branch and pushing to GitHub.
To https://github.com/gtauzin/kedro-dagster
   6118dfb..4738f70  gh-pages -> gh-pages
INFO    -  Your documentation should shortly be available at: https://gtauzin.github.io/kedro-dagster/
```

If anyone has any idea about what is happening, this would be greatly appreciated. Thank you!

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @gtauzin,
your current [website](https://gtauzin.github.io/kedro-dagster/) looks like it was created using Jekyll, so it seems you've selected the `main` branch with `/docs` directory as the source.
It should use the `gh-pages` branch and `/` root directory.
![image](https://github.com/user-attachments/assets/fa44f28b-959d-4722-8c40-74f808ddce6f)

The workflow pushes changes to the [`gh-pages`](https://github.com/gtauzin/kedro-dagster/tree/gh-pages) branch
Then GitHub per the configuration should detect changes and run another action, and in your case it run a Jekyll build:
- https://github.com/gtauzin/kedro-dagster/actions/workflows/pages/pages-build-deployment

**Reply by gtauzin** [reply]
Thank you, that makes sense!
